From 953e85fd77e209b7cd12d3640757b25b2100f372 Mon Sep 17 00:00:00 2001
From: Alex Marginean <alexandru.marginean@nxp.com>
Date: Fri, 4 Jan 2019 12:45:51 +0200
Subject: [PATCH 429/706] net: mscc: FELIX_SWITCH_TSN depends on TSN

This fixes a compilation issue with TSN disabled.

Signed-off-by: Alex Marginean <alexandru.marginean@nxp.com>
(cherry picked from commit 4074926014ff44a999f7123e4355581f55271209)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 drivers/net/ethernet/mscc/Kconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/net/ethernet/mscc/Kconfig b/drivers/net/ethernet/mscc/Kconfig
index 57648fe58fac..006b7d0fa37f 100644
--- a/drivers/net/ethernet/mscc/Kconfig
+++ b/drivers/net/ethernet/mscc/Kconfig
@@ -38,6 +38,7 @@ config MSCC_FELIX_SWITCH
 config MSCC_FELIX_SWITCH_TSN
 	tristate "TSN on FELIX switch driver"
 	depends on MSCC_FELIX_SWITCH
+	depends on TSN
 	help
 	  This driver supports TSN on felix switch.
 
-- 
2.17.1

